source/megaphone/main.cpp

```diff
@@ -0,0 +1,26 @@
+#include <iostream>
+#include <libphone.hpp>
+
+#include <spdlog/spdlog.h>
+#include <zenohc.hxx>
+
+auto main(int argc, char** argv) -> int {
+
+	zenohc::Config config;
+	config = zenohc::expect(zenohc::config_from_file(argv[1]));
```

Comment:
Use environment variable, not argument.

Reply:
that is also to be changed before the stable release.